src/admin/components/ResourceTabs/InformationTab/components/ReportInformationAside/QuickActions.tsx

```diff
@@ -49,7 +49,11 @@ const QuickActions: FC<QuickActionsProps> = ({ type }) => {
           <Button
             variant="outlined"
             component={Link}
-            to={createPath({ resource: modules.project.ResourceName, type: "show", id: record.project.uuid })}
+            to={createPath({
+              resource: modules.project.ResourceName,
+              type: "show",
+              id: record?.project?.uuid ?? record?.projectUuid
+            })}
```

Comment:
This will be needed eventually, but it shouldn't be needed for this PR. This component is only used for report types.

Reply:
yes, reverting